src/style/aframe.css

```diff
@@ -242,6 +242,96 @@ a-scene audio {
   display: block;
 }
 
+.a-device-motion-permission-full{
```

Comment:
I would probably made it a [gray opaque background](https://user-images.githubusercontent.com/39342/68625009-fcb6ea80-048c-11ea-9142-bdf80987620c.png) covering the whole scene with text and two big buttons. Smaller dialog / window partially covering the scene is harder to read in particular in iframed or embedded mode.

Reply:
Agreed; this is done in current commit.